From: Debian Rust Maintainers <pkg-rust-maintainers@alioth-lists.debian.net>
Date: Thu, 14 Jul 2022 13:17:39 +0200
Subject: d-test-ignore-avx-44056

Bug: https://github.com/rust-lang/rust/pull/55667
Forwarded: not-needed

===================================================================
---
 tests/ui/issues/issue-44056.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tests/ui/issues/issue-44056.rs
+++ b/tests/ui/issues/issue-44056.rs
@@ -1,5 +1,5 @@
 //@ build-pass (FIXME(55996): should be run on targets supporting avx)
-//@ only-x86_64
+//@ ignore-test
 //@ no-prefer-dynamic
 //@ compile-flags: -Ctarget-feature=+avx -Clto
 //@ ignore-backends: gcc
